crypto/threads_win.c

```diff
@@ -675,8 +729,13 @@ int CRYPTO_atomic_load_int(int *val, int *ret, CRYPTO_RWLOCK *lock)
 
     return 1;
 #else
-    /* On Windows, LONG is always the same size as int. */
+    /* On Windows, LONG is always the same size as int, */
+    /* except for the mingw64 target where long int is 64 bit. */
+# ifdef __MINGW64__
+    *ret = (int)InterlockedOr((__LONG32 volatile *)val, 0);
+# else
     *ret = (int)InterlockedOr((LONG volatile *)val, 0);
+# endif
```

Comment:
Given what @kwkam wrote above, this change is unnecessary and should be dropped.

Reply:
@t8m 
Done. I changed the comment as you suggested later
```
On Windows, LONG (but not long) is always the same size as int.
```